docs/getting_started/installation.md

```diff
@@ -180,6 +180,337 @@ schema_name - irods_environment
 schema_version - v3
 ~~~
 
+## Unattended Installation
+
+iRODS can also be installed by providing a file matching the JSON schema found here:
+
+- [https://github.com/irods/irods/blob/main/schemas/configuration/v4/unattended_installation.json.in](https://github.com/irods/irods/blob/main/schemas/configuration/v4/unattended_installation.json.in)
+
+This form of installation is known as an **Unattended Install**. Using this form of setup can help with automated deployments.
+
+To perform an **Unattended Install**, you'd run the following:
+```bash
+$ (sudo) python3 /var/lib/irods/scripts/setup_irods.py --json_configuration_file /path/to/your/json/input/file
+```
+
+Below is an example of what the input file for an iRODS Catalog Provider might contain:
```

Comment:
could make a `### Catalog Service Provider` here...

and a `### Catalog Service Consumer` down below... could make the nav a bit easier to see.